### PR TITLE
Update the change log for v0.6.0

### PR DESCRIPTION
**Pull request by mitalipaygude**
*Issue #, if available:*

*Description of changes:*
Update the change log for v0.6.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
Skipping CI for Draft Pull Request.
If you want CI signal for your change, please convert it to an actual PR.
You can still manually trigger a test run with `/test all`

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/535#" title="Author self-approved">mitalipaygude</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[docs/OWNERS](https://github.com/aws/eks-anywhere/blob/main/docs/OWNERS)~~ [mitalipaygude]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: docs/content/en/docs/reference/changelog.md
Comment (mdsgabriel) [inline docs/content/en/docs/reference/changelog.md:15]: I believe this could be 

```
Support to create and manage workload clusters
```

@jaxesn @vipin-mohan ?

File: docs/content/en/docs/reference/changelog.md
Comment (rothgar) [inline docs/content/en/docs/reference/changelog.md:13]: Do we have tasks or concepts documented for any of these features? It would be great to be able to link readers back to our docs on getting started with each new feature.

File: docs/content/en/docs/reference/changelog.md
Comment (TerryHowe) [inline docs/content/en/docs/reference/changelog.md:4]: I think we need the header so this shows up in the TOC
Screen Shot 2021-11-01 at 14 55 21

Not sure why it was removed

Comment (mitalipaygude) [reply docs/content/en/docs/reference/changelog.md:4]: was removed by mistake! added it back.

File: docs/content/en/docs/reference/changelog.md
Comment (chrisnegus) [reply docs/content/en/docs/reference/changelog.md:13]: > Do we have tasks or concepts documented for any of these features? It would be great to be able to link readers back to our docs on getting started with each new feature.

I'll be writing up content related to the "management cluster". I know there are also docs on the "bring your own registry" feature and possibly for the upgrade improvements. I agree we should add links to those docs here once they are locked down.

Comment (mitalipaygude) [reply docs/content/en/docs/reference/changelog.md:13]: yes we do, I was deciding on whether we should add the issue or the prs' links. I will add the issue and docs links.

File: docs/content/en/docs/reference/changelog.md
Comment (abhinavmpandey08) [inline docs/content/en/docs/reference/changelog.md:26]: I would move this to the "Fixed" section since it's not a new feature but a fix for a pre-existing feature

File: docs/content/en/docs/reference/changelog.md
Comment (abhinavmpandey08) [inline docs/content/en/docs/reference/changelog.md:25]: fix broken link
```suggestion
- Make namespace configurable for anywhere resources [#177](https://github.com/aws/eks-anywhere/pull/177/files)
```

File: docs/content/en/docs/reference/changelog.md
Comment (abhinavmpandey08) [inline docs/content/en/docs/reference/changelog.md:24]: I would say "Ability to use your own container registry" instead
```suggestion
- Ability to use your own container registry [#98](https://github.com/aws/eks-anywhere/issues/98)
```

**Review by abhinavmpandey08 — APPROVED** [review]

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/535#pullrequestreview-795848748" title="Approved">abhinavmpandey08</a>*, *<a href="https://github.com/aws/eks-anywhere/pull/535#" title="Author self-approved">mitalipaygude</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[docs/OWNERS](https://github.com/aws/eks-anywhere/blob/main/docs/OWNERS)~~ [abhinavmpandey08,mitalipaygude]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: docs/content/en/docs/reference/changelog.md
Comment (chrisnegus) [inline docs/content/en/docs/reference/changelog.md:4]: I'd like to add a link to this page to the menu bar. I think changing the header to look like this would work:
`title: "What's New?"
linkTitle: "What's New?"
weight: 55
menu:
  main:
    weight: 10
---
`

Comment (chrisnegus) [reply docs/content/en/docs/reference/changelog.md:4]: So this would replace the two lines of three-dashes and everything in between.

File: docs/content/en/docs/reference/changelog.md
Comment (chrisnegus) [inline docs/content/en/docs/reference/changelog.md:20]:
```suggestion
- Support for upgrading eks-anywhere components [#93](https://github.com/aws/eks-anywhere/issues/93), [cluster upgrades] [Create cluster workflow]({{< relref "/docs/tasks/cluster/cluster-upgrades" >}})
```
This form of links gets checked by hugo.

File: docs/content/en/docs/reference/changelog.md
Comment (chrisnegus) [inline docs/content/en/docs/reference/changelog.md:2]: Sorry, I think I wasn't clear before. You can remove lines 2, 3, and 4, I think.

**Pull request merged**
